### PR TITLE
changed enterprise dashboard to enterprise console

### PR DESCRIPTION
**Pull request by RynhardPietersen**
## Summary

What started out as Enterprise Dashboard is more than Dashboard now, as it includes Enterprise Settings, Billings Statements and soon to be SMS Detail Billings and Manage SSO.
So, a proper name for this module should be Enterprise Console.

Link to ticket: https://dimagi-dev.atlassian.net/browse/SS-145

## Product Description

Users will now see Enterprise Console instead of Enterprise Dashboard

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan

This was tested locally and will be tested on staging but it's literally just a name change

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Review by mjriley — APPROVED** [review]
Looks good. Approving as the tests pass and the name changes look good, but I don't have any context if there might be side-effects to this (like if any `dashboard` names are stored in the database).

**Comment by RynhardPietersen** [comment]
> Looks good. Approving as the tests pass and the name changes look good, but I don't have any context if there might be side-effects to this (like if any `dashboard` names are stored in the database).

I tested and it seemed like everything worked out. Would this be something that needs QA?

**Review by biyeun — CHANGES_REQUESTED** [review]
please only do a wording change of user-facing text to keep things simple as there's a lot of development with ERM still underway and it's not necessary for the backend to be refactored right now to reflect the naming change.

**Comment by RynhardPietersen** [comment]
I removed the commit that changed the names of all the urls, method names and files.

File: corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ hqDefine("enterprise/js/enterprise_dashboard", [
     kissmetrics
 ) {
     $(function () {
-        kissmetrics.track.event("[Enterprise Dashboard] Visited page");
+        kissmetrics.track.event("[Enterprise Console] Visited page");
```
Comment (mjriley) [inline corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js:14]: @biyeun is this a safe change? I do think its correct to switch over as much as possible, but I want to make sure we don't have any scripts or whatever that depend on event names

Comment (biyeun) [reply corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js:14]: yeah, I would not change this. just the user-facing text as the ticket asks @RynhardPietersen

File: corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ hqDefine("enterprise/js/enterprise_dashboard", [
             });
 
             $element.find(".btn-primary").click(function () {
-                kissmetrics.track.event("[Enterprise Dashboard] Clicked Email Report for " + slug);
+                kissmetrics.track.event("[Enterprise Console] Clicked Email Report for " + slug);
```
Comment (biyeun) [inline corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js:28]: please revert this change too, thank you! @RynhardPietersen

**Review by biyeun — CHANGES_REQUESTED** [review]
a couple more small changes and then this is g2g

**Comment by biyeun** [comment]
@RynhardPietersen it looks like after re-reading the ticket, we are keeping the "Enterprise Dashboard" page. It's only the top-level section that is being renamed to "Enterprise Console". I will add some additional comments

File: corehq/apps/enterprise/tasks.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def email_enterprise_report(domain, slug, couch_user):
     # Send email
     url = absolute_reverse("enterprise_dashboard_download", args=[domain, report.slug, str(hash_id)])
     link = "<a href='{}'>{}</a>".format(url, url)
-    subject = _("Enterprise Dashboard: {}").format(report.title)
+    subject = _("Enterprise Console: {}").format(report.title)
```
Comment (biyeun) [inline corehq/apps/enterprise/tasks.py:39]: please revert this change. should remain enterprise dashboard as we are just changing the section name

File: corehq/apps/enterprise/tests/test_enterprise_tasks.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def test_email_report_domains_successful(self, mock_send):
         with patch('corehq.apps.enterprise.tasks.get_redis_client', return_val=mock_redis_client):
             email_enterprise_report(self.domain, EnterpriseReport.DOMAINS, self.couch_user)
 
-        expected_title = "Enterprise Dashboard: Project Spaces"
+        expected_title = "Enterprise Console: Project Spaces"
```
Comment (biyeun) [inline corehq/apps/enterprise/tests/test_enterprise_tasks.py:46]: please revert all the changes in this file. should remain enterprise dashboard as we are just changing the section name

File: corehq/apps/enterprise/views.py [hunk 5]
```diff
@@ -66,8 +66,8 @@ def enterprise_dashboard(request, domain):
             EnterpriseReport.FORM_SUBMISSIONS,
         )],
         'current_page': {
-            'page_name': _('Enterprise Dashboard'),
-            'title': _('Enterprise Dashboard'),
+            'page_name': _('Enterprise Console'),
```
Comment (biyeun) [inline corehq/apps/enterprise/views.py:69]: please revert this change and the line below. we want to keep the page name the same "Enterprise Dashboard". just the section should change

File: corehq/apps/enterprise/views.py [hunk 6]
```diff
@@ -157,7 +157,7 @@ def edit_enterprise_settings(request, domain):
 
 @method_decorator(require_enterprise_admin, name='dispatch')
 class BaseEnterpriseAdminView(BaseDomainView):
-    section_name = ugettext_lazy("Enterprise Dashboard")
+    section_name = ugettext_lazy("Enterprise Console")
```
Comment (biyeun) [inline corehq/apps/enterprise/views.py:160]: this is good. keep this ✅

File: corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html [hunk 7]
```diff
@@ -37,7 +37,7 @@
                 <li class="dropdown-header nav-header">{% trans 'Manage Enterprise' %}</li>
                 <li>
                     <a href="{% url "enterprise_dashboard" domain %}">
-                        <i class="fa fa-cogs icon-cogs dropdown-icon"></i> {% trans "Enterprise Dashboard" %}
+                        <i class="fa fa-cogs icon-cogs dropdown-icon"></i> {% trans "Enterprise Console" %}
```
Comment (biyeun) [inline corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html:40]: this is good. keep this ✅

File: corehq/tabs/tabclasses.py [hunk 8]
```diff
@@ -1555,7 +1555,7 @@ def sidebar_items(self):
         if has_privilege(self._request, privileges.PROJECT_ACCESS):
             enterprise_views.extend([
                 {
-                    'title': _('Enterprise Dashboard'),
+                    'title': _('Enterprise Console'),
```
Comment (biyeun) [inline corehq/tabs/tabclasses.py:1558]: revert please

File: corehq/apps/sso/forms.py [hunk 9]
```diff
@@ -258,7 +258,7 @@ class EditIdentityProviderAdminForm(forms.Form):
         )
     )
     is_editable = forms.BooleanField(
-        label=ugettext_lazy("Enterprise Dashboard"),
+        label=ugettext_lazy("Enterprise Console"),
```
Comment (biyeun) [inline corehq/apps/sso/forms.py:261]: this is good. keep this ✅

File: corehq/apps/accounting/models.py [hunk 10]
```diff
@@ -2851,7 +2851,7 @@ def email_context(self):
         payment_status = (_("Paid")
                           if self.invoice.is_paid or self.invoice.balance == 0
                           else _("Payment Required"))
-        # Random domain, because all subscriptions on a customer account link to the same Enterprise Dashboard
+        # Random domain, because all subscriptions on a customer account link to the same Enterprise Console
```
Comment (biyeun) [inline corehq/apps/accounting/models.py:2854]: this is good ✅  keep

File: corehq/apps/domain/models.py [hunk 11]
```diff
@@ -287,7 +287,7 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
         see data, reports, charts, etc.
 
         Exceptions: accounting has some models that combine multiple domains,
-        which make "enterprise" multi-domain features like the enterprise dashboard possible.
+        which make "enterprise" multi-domain features like the enterprise console possible.
```
Comment (biyeun) [inline corehq/apps/domain/models.py:290]: this is good ✅  keep

File: corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js (hunk 1 above)
Comment (biyeun) [reply corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js:14]: Ah ok, so I think there was a slight misunderstanding of the ticket. We want to rename the top-level section to "Enterprise Console" to make it distinct from the actual "Enterprise Dashboard" page. Keep the Enterprise Dashboard page as is...we are not intending to do a refactor here at all.

**Review by biyeun — APPROVED** [review]
thank you! 🎉
